BuildSourceImage.sh

```diff
@@ -1,80 +1,968 @@
-#!/bin/sh
+#!/bin/bash
 
-# This script requires an OCI IMAGE Name to pull.
-# The script generates a SOURCE Image based on the OCI Image
-# Script must be executed on the same OS or newer as the image.
-if test $# -lt 2 ; then
-    echo Usage: $(basename $0) IMAGE CONTEXT_DIR [EXTRA_SRC_DIR]
+# This script builds a Source Image via "drivers" to collect source
+
+export ABV_NAME="SrcImg"
+# TODO maybe a flag for this?
+export source_image_suffix="-source"
+
+
+_usage() {
+    echo "Usage: $(basename $0) [-D] [-b <path>] [-c <path>] [-e <path>] [-r <path>] [-o <path>] [-i <image>] [-p <image>] [-l] [-d <drivers>]"
```

Comment:
```
In BuildSourceImage.sh line 11:                                                                                                                
    echo "Usage: $(basename $0) [-D] [-b <path>] [-c <path>] [-e <path>] [-r <path>] [-o <path>] [-i <image>] [-p <image>] [-l] [-d <drivers>]"
                            ^-- SC2086: Double quote to prevent globbing and word splitting.                                                   
```